ftl/node/main.py

```diff
@@ -96,6 +108,18 @@ def main(args):
             logging.info('Generating app layer...')
             app_layer, diff_id = bldr.BuildAppLayer()
             with append.Layer(deps, app_layer, diff_id=diff_id) as app_image:
+                if args.local:
+                    if not args.filepath:
+                        raise AssertionError('Please provide path to tar \
+                                             with --filepath flag')
+                    dir = os.path.dirname(args.filepath)
```

Comment:
You could leave off this check and just assume the user created a tar ahead of time.

Reply:
We might also want to swap out or disable the cache if --local mode is used. WDYT?

Reply:
I think disabling it could make sense, but what would we swap it out for?